aboutus.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./aboutus.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat&display=swap" rel="stylesheet">
+</head>
+<body>
+    <div class="about_text">
+        <p class="about_text_title">-Our Mission-</p>
+        <p class="about_p" style="text-align: center;">
+            To provide the most actionable Real Estate Website.
+        </p>
+        <p class="about_text_title">About</p>
+
+        <p class="about_p">
```

Comment:
`about_p` What this class name means

**Always use descriptive class names, that tell us what is the content**